cypress/integration/examples/network_requests.spec.js

```diff
@@ -0,0 +1,201 @@
+/// <reference types="cypress" />
+
+context('Network Requests', () => {
+  beforeEach(() => {
+    cy.visit('https://example.cypress.io/commands/network-requests')
+  })
+
+  // Manage AJAX / XHR requests in your app
+
+  it('cy.server() - control behavior of network requests and responses', () => {
```

Comment:
The test cases should be related to the Book Store app, covering login, add book, and delete book functionality. Replace the current test cases with appropriate ones.